### ISSUE TITLE
ESSFM related Discord.

### ISSUE THREAD

**Issue by pachuco** [jwt27/esfm#4]
If you're into that sort of thing, I make a discord around this chipset.

Basically, I think things will become easier if people with same interest are in same place.
New link: https://discord.gg/r9YXEgQjZr

**Comment by jwt27** [comment]
Thanks, will join.  Although I was hoping we could keep all technical discussion in this github repo - discord is not great for searching.

**Comment by pachuco** [comment]
Agreed on the unsearchability. But it is faster for discussing and getting people interested!
For best of both worlds, use both.

**Comment by jwt27** [comment]
Good point.  I added the invite link in the "about" section on this repo.

**Comment by mlfatr** [comment]
Hey there,
I'm Malfunction/Altair - could I ask for Discord access? I'm more than curious about ESFM for years already :)

**Comment by pachuco** [comment]
> Hey there, I'm Malfunction/Altair - could I ask for Discord access? I'm more than curious about ESFM for years already :)

link in OP post. Unless it expired.

**Comment by mlfatr** [comment]
Expired already, unfortunately :(

Cheers,
Slawek

sob., 29 lip 2023 o 06:26 pachuco ***@***.***> napisał(a):

**Comment by pachuco** [comment]
Eh, Discord disabled permanent invites unless you turn into a "community server"(verified email + scan messages). That sucks.

https://discord.gg/mRrdkvE3j

**Comment by wbcbz7** [comment]
could you please update the invitation link (if that discord group is still relevant)? seems like it's expired again

**Comment by pachuco** [comment]
try https://discord.gg/r9YXEgQjZr

**Comment by pachuco** [comment]
The above discord link still works, but I am moving shop to https://matrix.to/#/#essfm-shenanigans:matrix.org
ESFMu dev has left for Matrix, and in light of Discord's latest faux pas, I am going there too.
